lib/src/persistence/persistence_error.dart

```diff
@@ -0,0 +1,5 @@
+class PersistenceError extends Error {
```

Comment:
Could you please add documentation for this class?

Reply:
Done.